### ISSUE TITLE
[Bug]: After installation issue, Error code on class Flatpickr and couple of things i saw

### ISSUE THREAD

**Issue by dharen008** [savannabits/filament-flatpickr#66]
### What happened?

1. After installing and adding this code:

```
use Coolsam\FilamentFlatpickr\Forms\Components\Flatpickr;

Flatpickr::make( 'date' )
```

I encountered a 'class not found' error. Upon inspecting the Flatpickr class file, I found an error on line 502:

```
    public function minDate(Carbon|string|null $minDate|Closure): static
    {
        $this->minDate = $minDate ? Carbon::parse($minDate) : $minDate;

        return $this;
    }
```

### How to reproduce the bug

I copied the same format for maxDate and changed it to minDate, and now it's fixed:

```
    public function minDate(Carbon|string|null|Closure $minDate = 'now'): static
    {
        $this->minDate = $minDate ? Carbon::parse($minDate) : $minDate;

        return $this;
    }
```

after i fix the issue. the frontend is working but on the resource the unknown class is still there

![image](https://github.com/savannabits/filament-flatpickr/assets/109984612/a4e98138-38f2-435b-80be-ba62a6fa1628)

### What happened?

2. When your database column type is on time only and using altFormat & altInput will not work it will not auto convert the data value,
it will only work on timestamp with date and time on the value

![image](https://github.com/savannabits/filament-flatpickr/assets/109984612/64c71f3e-4d9f-4297-86eb-1f02d041aa2a)

```
Flatpickr::make( 'time_in' )
    ->altFormat( 'h:i K' )
    ->noCalendar( true )
    ->altInput( true )
    ->enableTime(),
```

### What happened?

3. When using ->altInputClass('sample-class') it will replace all the class on that input class, is there a way to just add that class to the input class

```
// Resource file
Flatpickr::make( 'time_in' )
    ->altInputClass( 'sample-class' )
    ->altFormat( 'h:i K' )
    ->noCalendar( true )
    ->altInput( true )
    ->enableTime(),
   
// AppProvider File
Flatpickr::configureUsing( function ( Flatpickr $flatinput ): void {
     $flatinput->altInputClass( 'sample-class' );
} );
```

### What happened?

4. When using the Flatpickr date picker component to display and select dates with time, there's a discrepancy between the data stored in the database and the output shown to the user.

Database: 2024-05-17 00:00:00 
Output: May 17, 2024 8:00 AM

```
Flatpickr::make('created_at')
    ->altInput(true)
    ->altFormat('F d, Y h:i K')
    ->enableTime(),
```

### Package Version

3.1

### PHP Version

8.2

### Laravel Version

11

### Which operating systems does with happen with?

Windows

**Comment by davisngl** [comment]
Seems that [this](https://github.com/savannabits/filament-flatpickr/commit/5fabe1cdbee6d2b215d5139ad0166cfec075a3d8) bug fix did not get into this minor release, which resulted in this still happening.

**Comment by dharen008** [comment]
saw another issue when trying to remove

![image](https://github.com/savannabits/filament-flatpickr/assets/109984612/7cef31a3-5f55-46e0-9be5-df6f1649d8d9)

**Comment by HUHAO-mll** [comment]
I get the same issue, have you resolved it? Is there any alternative plugin?

**Comment by dharen008** [comment]
> I get the same issue, have you resolved it? Is there any alternative plugin?

I stopped using this plugin because it has many issues that need fixing. The DatePicker->native(false) already handles many flatpicker functions, so I decided to stick with the default Filament DatePicker.

**Comment by coolsam726** [comment]
Fixed in #58
